### PR TITLE
Kkotorov/redesign

### PR DESCRIPTION
**Pull request by Mike4o**

### CODE REVIEW

**Comment by teobugslayer** [comment]
Perhaps "Device Version" should be "OS Version", otherwise, :+1:

**Comment by teobugslayer** [comment]
On the bottom of the screen there are the AppBuilder logo and the text "Feature Example". Perhaps we can make them tappable and open browser to the platform and to the samples.

**Comment by Mike4o** [comment]
I am not sure if opening the samples page on a device will be of any help to our users.

**Comment by didiepetrova** [comment]
In ReadMe.md, alt and title should be:  alt="Try in the Platform" title="Try in the Platform"

**Comment by didiepetrova** [comment]
In index.html, ensure that there is a viewport setting: <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />.

**Comment by didiepetrova** [comment]
:+1: p.s. Works and looks good. There is a small hover on the top right of the screenshot

File: sample-device/index.html [hunk 1]
```diff
@@ -1,33 +1,49 @@
...
-	<footer class="footer">
-		<div>Cordova API Samples</div>
-	</footer>
+                <div class="col">
+                    <button class="button" id="deviceCordovaVersion">Cordova version</button>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col">
+                    <button class="button" id="devicePlatform">Device patform</button>
```
Comment (TsvetinaIvanova) [inline sample-device/index.html]: Device p**l**atform

**Comment by TsvetinaIvanova** [comment]
:+1: after you fix the typo.

**Pull request merged**